website/docs/cheatsheets/python/v1/troubleshooting.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Troubleshooting
+id: troubleshooting
+description: A cheat sheet for Azure ML.
+keywords:
+  - azure machine learning
+  - aml
+  - troubleshooting
+---
+
+### Error: az acr login- APIVersion 2020-11-01-preview is not available. 
+**Description**
+I built a mlnet image successfully using Azure Machine Learning compute instance last week. However, I got the error when rerun the same notebook. 
```

Comment:
Can we make this description more generic. It seems to be an issue with `az acr login`, so can we frame it in that level of generality.